bundles/org.openhab.binding.mikrotik/README.md

```diff
@@ -0,0 +1,374 @@
+# Mikrotik RouterOS Binding
+
+This binding integrates [Mikrotik](https://mikrotik.com/) [RouterOS](https://help.mikrotik.com/docs/display/ROS/RouterOS)
+[devices](https://mikrotik.com/products) allowing monitoring of system resources, network interfaces and WiFi clients.
+
+## Supported Things
+
+* `routeros` - An instance of the RouterOS device connection
+* `interface` - A network interface inside RouterOS device
+* `wifiRegistration` - Any wireless client connected to a RouterOS wireless network (regular or CAPsMAN-managed)
+
+
+## Discovery
+
+Discovery is currently not supported, but may be implemented in future versions.
+
+
+## Binding Configuration
+
+The binding has no configuration options, all configuration is done at the Bridge and Thing levels.
+
+
+## Bridge Configuration
+
+To use this binding you need at least one RouterOS-powered device (Bridge) accessible to the host running
+OpenHAB via network.
+Make sure your RouterOS have API enabled by visiting [<kbd>IP -> Services</kbd>](https://wiki.mikrotik.com/wiki/Manual:IP/Services) 
+configuration section in 
+[WinBox](https://wiki.mikrotik.com/wiki/Manual:Winbox). Note the API port number as you'll need it below.
+[SSL API connection](https://wiki.mikrotik.com/wiki/Manual:API-SSL) is not yet supported by this binding.
+To connect to RouterOS API you need to provide user credantials to the bridge config. You may use your current 
+credentials you use to manage device, but it is highly recommended to **create a read-only RouterOS user** since
+this binding only reads data from the device. To do so, proceed to <kbd>System -> Users</kbd> configuration section
+and add a user to the `read` group.
+
+> Thing type: `routeros`
+
+The RouterOS Bridge configuration parameters are:
+
+* `host` _(required text)_ [Default: `192.168.88.1`] - Hostname or IP address of the RouterOS device
+* `port` _(optional integer)_ [Default: `8729`] - API Port number of the RouterOS device
+* `login` _(required text)_ [Default: `admin`] - The user login to access the the RouterOS device
+* `password` _(required text)_ - The user password to access the RouterOS device
+* `refresh` _(optional integer)_ [Default: `10`] - The refresh interval in seconds to poll the RouterOS device
+
+**All things provided by this binding requires a working bridge to be set up.**
+
+
```

Comment:
Two empty lines in a row, I'm sure this is one of the auto checks done. Please compile and then look inside your TARGET folder where you will find a code analasis folder. In that folder is a report of things to address. Please go through that and any warnings that show up when you compile if waiting for more review comments.

Reply:
This is what it shows currently
![image](https://user-images.githubusercontent.com/81665/111929682-be3d3100-8abf-11eb-99fa-5e0f900796d0.png)

@Skinah please confirm I should fix all those 15 markdown issues.

Reply:
Yes please fix.